homeassistant/components/incomfort/__init__.py

```diff
@@ -53,3 +56,40 @@
         )
 
     return True
+
+
+class IncomfortEntity:
```

Comment:
We should inherit from `Entity` since we're implementing entity properties.

Reply:
Sorted.